core/templates/dev/head/pages/community-dashboard-page/community-dashboard-page.directive.html

```diff
@@ -47,10 +47,10 @@
   <div ng-if="$ctrl.activeTabName === 'myContributionTab' || !$ctrl.activeTabName">
     <contributions-and-review></contributions-and-review>
   </div>
-  <div ng-if="$ctrl.activeTabName === 'questionTab'">
+  <div ng-if="$ctrl.activeTabName === 'submitQuestionTab'">
```

Comment:
Thanks for fixing this I think this happened because of the race condition. We should start writing e2e test to avoid such things to happen in the future.

Reply:
Got it.